pkg/cmd/roachprod-microbench/main.go

```diff
@@ -89,7 +89,8 @@ func makeRunCommand() *cobra.Command {
 	cmd.Flags().StringVar(&config.remoteDir, "remote-dir", config.remoteDir, "working directory on the target cluster")
 	cmd.Flags().StringVar(&config.timeout, "timeout", config.timeout, "timeout for each benchmark e.g. 10m")
 	cmd.Flags().StringVar(&config.shellCommand, "shell", config.shellCommand, "additional shell command to run on node before benchmark execution")
-	cmd.Flags().StringSliceVar(&config.excludeList, "exclude", []string{}, "comma-separated regex of packages and benchmarks to exclude e.g. 'pkg/util/.*:BenchmarkIntPool,pkg/sql:.*'")
+	cmd.Flags().StringSliceVar(&config.excludeList, "exclude", []string{}, "benchmarks to exclude, in the form <pkg regex:benchmark regex> e.g. 'pkg/util/.*:BenchmarkIntPool,pkg/sql:.*'")
+	cmd.Flags().StringSliceVar(&config.noProbePkgList, "no-probe", []string{}, "packages to completely exclude from probing or execution'")
```

Comment:
Nit: `no-probe` is kinda awkward; why not `ignore-packages`?

Reply:
Yep, naming was not my strong suite today. `ignore-packages` is clearer.